cmd/protoc-gen-gorums/gengorums/template_server.go

```diff
@@ -12,9 +12,9 @@ var serverInterface = `
 type {{$service}} interface {
 	{{- range .Methods}}
 	{{- if isOneway .}}
-	{{.GoName}}({{$context}}, *{{in $genFile .}})
+	{{.GoName}}(ctx {{$context}}, request *{{in $genFile .}})
 	{{- else}}
-	{{.GoName}}({{$context}}, *{{in $genFile .}}, func(*{{out $genFile .}}, error))
+	{{.GoName}}(ctx {{$context}}, request *{{in $genFile .}}) (response *{{out $genFile .}}, err error)
```

Comment:
Any particular reason you want to name the signature types?

Reply:
I think it makes it easier to understand the interface if you look through the generated code. Also, when implementing the server interfaces, I usually use the ["Generate Interface Stubs"](https://github.com/golang/vscode-go/blob/master/docs/features.md#generate-interface-implementation) command from vscode-go to auto generate the methods. When the parameters are named, the names get filled in automatically.